src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -317,17 +317,19 @@ export const WorkspaceListInner = ({
     };
 
     return (
-      <>
-        <EuiButton color="danger" iconType="trash" onClick={onClick}>
-          Delete {selection.length} Workspace
-        </EuiButton>
-        {deletedWorkspaces && deletedWorkspaces.length > 0 && (
-          <DeleteWorkspaceModal
-            selectedWorkspaces={deletedWorkspaces}
-            onClose={() => setDeletedWorkspaces([])}
-          />
-        )}
-      </>
+      isDashboardAdmin && (
+        <>
+          <EuiButton color="danger" iconType="trash" onClick={onClick}>
+            Delete {selection.length} Workspace
+          </EuiButton>
+          {deletedWorkspaces && deletedWorkspaces.length > 0 && (
+            <DeleteWorkspaceModal
+              selectedWorkspaces={deletedWorkspaces}
+              onClose={() => setDeletedWorkspaces([])}
+            />
+          )}
+        </>
+      )
     );
```

Comment:
should we hide the checkbox as well?  the only action user can take for multiple select is batch delete, if we hide the delete button and keep multi select, that looks a little wired user can't do anything after select.